Framework/PythonInterface/test/python/mantid/plots/plots__init__Test.py

```diff
@@ -472,6 +473,35 @@ def test_that_relim_ignores_interactive_markers(self):
         ax.autoscale()
         np.testing.assert_almost_equal((y_min, y_max), ax.get_ylim())
 
+    def test_converting_to_and_from_waterfall_plot(self):
```

Comment:
This test could be split into parts, such as converting to waterfall, filling area under curve, removing filled area, converting back...